src/PKSim.Presentation/Presenters/Parameters/TableParameterPresenter.cs

```diff
@@ -143,4 +148,28 @@ private ValuePoint valuePointFrom(ValuePointDTO valuePointDTO)
 
       public override void RemovePoint(ValuePointDTO pointToRemove) => _tableFormulaDTO.AllPoints.Remove(pointToRemove);
    }
+
+   public class TableParameterPresenter : TableParameterPresenter<ITableFormulaView>, ITableParameterPresenter
+   {
+      public TableParameterPresenter(ITableFormulaView view, IParameterTask parameterTask, IFormulaFactory formulaFactory, ICloner cloner) :
+         base(view, parameterTask, formulaFactory, cloner)
+      {
+         // default import function disabled when context is not specified
+         view.ImportVisible = CanImport;
+      }
+
+      protected override DataRepository ImportTablePoints()
+      {
+         // default import function disabled when context is not specified
+         throw new System.NotImplementedException();
```

Comment:
NotImplementedException?

Reply:
Yeah, the interface requires these methods, but to satisfy the button for import, however, in this case the button is hidden with the comment that import is not allowed. We could return DataRepos and TableFormula or return null to avoid this ugly sight